clock_exercise/TimerSettingPage.qml

```diff
@@ -50,15 +50,15 @@ Rectangle {
         font.styleName: "Regular"
     }
 
-    StartButton {
+    BigButton {
         x: 23; y: 706
+        enabled: (hoursNumberField.value === 0 && minutesNumberField.value === 0)? false: true
```

Comment:
Come avrai capito non sono un grande fan dell'operatore ternario! :P
Anzi, in realtà mi piace, ma cerco di evitarlo quando finisce solo per "allungare il brodo", ovvero in tutti quei casi in cui:
* i valori ritornati dall'operatore ternario sono `true`:`false` o viceversa
* i valori ritornati dall'operatore ternario sono `1`:`0` o viceversa
* in generale se i valori ritornati sono immediatamente deducibili o semplificabili a partire dalla condizione del ternario stesso

In questo caso ad esempio propenderei per:
`enabled: hoursNumberField.value || minutesNumberField.value`